ucey-star/ID8
# Adds and Adjusts Personal Details and Project Idea

## Description

In this PR: 
1. I removed the Guidelines Page, which we are not using anymore (given the V2 of the High Fidelity Design). 
2. I updated the Personal Details page after the updated increments in the design. 
3. Added the Project Idea page, with a few differences from the design that I would like to discuss with the design team.
4. Added instructions in the README.md file to ensure we are able to push the code with appropriate formatting. 

## Type of Change


## Testing Instructions

Just run `npm run dev` and you will be able to see the updated pages in the web app. 

## Screenshots (if applicable)

**Personal Details Page**
<img width="685" alt="Screenshot 2024-11-12 at 7 43 29 AM" src="https://github.com/user-attachments/assets/15ba68ed-5512-4d4f-9fc8-0b407747e4ee">



**Project Idea Page**
<img width="631" alt="Screenshot 2024-11-12 at 7 43 50 AM" src="https://github.com/user-attachments/assets/ffa535c1-f97a-494c-bdbb-3de341f1dc9c">
